app/Decsys/Repositories/Contracts/IWebhookRepository.cs

```diff
@@ -9,7 +10,7 @@ public interface IWebhookRepository
     /// </summary>
     /// <param name="webhook">The webhook model to create.</param>
     /// <returns>The created Webhook Id</returns>
-    int Create(WebhookModel webhook);
+    ObjectId Create(WebhookModel webhook);
```

Comment:
Can't expose ObjectId outside the repository as it's Mongo specific. use `string` instead